ras/ras_extended.py

```diff
@@ -105,6 +105,9 @@ def test1_ls_vdev(self):
         self.run_cmd("ls-vdev")
         self.run_cmd("ls-vdev -h")
         self.run_cmd("ls-vdev -V")
+        dev_name = self.run_cmd_out("ls-vdev").split()[1]
+        if dev_name not in self.run_cmd_out("lsblk"):
```

Comment:
I feel for some outputs it will returns a False `True`.
like if dev_name is `sda` and `lscfg`  has sdaa or sdab etc which doesn't match exact device name but above if_loop will return true for this scenario.

So I think you can tweak it a little bit differently like,

```
for line in self.run_cmd_out("lsblk"):
    if dev_name != line
        self.is_fail += 1
```

Reply:
Naresh you are right but again with your logic it will not be valid to compare a line with a string

Line may contains other strings with device name

Also it has multiple lines, for every line which do not have the device name it fails
May something like

flag = True
      for line in self.run_cmd_out("lsblk"):
          if dev_name in line:
               if dev_name == line.split()[0]:
                   flag = False
                   break;
      if flag:
             self. is_fail += 1

Reply:
@shirishaganta1

Reply:
@abdhaleegit just reverse the logic, if found increase the count and if count is 0 the fail.